src/components/prompts/TermsAndConditions.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import {
+  Button,
+  Form,
+  Modal,
+  Radio,
+} from 'semantic-ui-react';
+
+/**
+ * Displays a model that requires the user to accept the terms and conditions before using the app
+ * @extends React.Component
+ * @param {boolean} termsAccepted - boolean indicating whether terms and conditions have been accepted by the user
+ * @param {function} toggleAcceptTerms - function to set the termsAccepted in app state
+ * @param {object} snippets - object containing localization snippets
+ */
+class TermsAndConditions extends Component {
+  
+  state = { canCountOnPredictions: null };
+
+  closeModal = (accept) => {
+    if (accept) {
+      this.props.toggleAcceptTerms();
+    } else {
+      this.props.history.push('/');
+    }
+  };
+
+  handleChange = (canCountOnPredictions) => {
+    this.setState({ canCountOnPredictions });
+  };
+
+  render() {
+  
+    const {
+      termsAccepted,
+      snippets,
+    } = this.props;
+
+    return (
+      <Modal
+        id={ `WarningModal` }
```

Comment:
Class names gets overwritten out semantic's classes so we' likely need to wrap the modal with a div.  I was trying to avoid adding that, hence using ID.  We need to figure out a better way of handling the overriding of classes/styles of semantic.  But tat shouldn't be part of this PR.

Reply:
You might be able to experiment with `#App .warning-modal ...` and such things to get specific. It's all a selector war one way or another. The inspector can help.